src/Uno.UI/Uno.UI.csproj

```diff
@@ -344,6 +344,14 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Remove="UI\Xaml\Controls\Flyout\FlyoutBase.net.cs" />
```

Comment:
...or these changes. No changes to this file. `Uno.UI.csproj` uses globbing (`*.blah.cs`), there shouldn't be explicit file references. (VS adds them incorrectly.)

Reply:
good to know :p